src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -378,6 +378,58 @@ def test_merged_sse_response(self, mock_read_write: pytest.MonkeyPatch) -> None:
         pd._preprocess_response(res_timestamps, res_outputs)
         assert res_outputs[1]["response"] == expected_response
 
+    def test_openai_output_token_counts(
+        self, mock_read_write: pytest.MonkeyPatch
+    ) -> None:
+        res_outputs = [
+            {
+                "response": 'data: {"choices":[{"delta":{"content":"this"}}],"object":"chat.completion.chunk"}\n\n'
+            },
+            {
+                "response": 'data: {"choices":[{"delta":{"content":" is"}}],"object":"chat.completion.chunk"}\n\n'
+            },
+            {
+                "response": 'data: {"choices":[{"delta":{"content":" a"}}],"object":"chat.completion.chunk"}\n\n'
+            },
+            {
+                "response": 'data: {"choices":[{"delta":{"content":" test"}}],"object":"chat.completion.chunk"}\n\n'
+            },
+        ]
+
+        tokenizer = get_tokenizer(DEFAULT_TOKENIZER)
+        pd = LLMProfileDataParser(
+            filename=Path("openai_profile_export.json"),
+            tokenizer=tokenizer,
+        )
+
+        output_token_counts, total_output_token = pd._get_output_token_counts(
+            res_outputs
+        )
+        assert output_token_counts == [1, 1, 1, 1]
```

Comment:
Can we update these tests to not all be 1 token?
In general, I have found that testing for 0, 1, 2+ usually provides enough coverage to avoid hiding assumptions. All 1's can hide something.

Reply:
Good point. Updated the test